src/reanimated/ReanimatedScreen.tsx

```diff
@@ -1,103 +1,25 @@
-import React, { PropsWithChildren } from 'react';
-import { Platform, View } from 'react-native';
-import {
-  Screen,
-  ScreenProps,
-  ScreenContext,
-  TransitionProgressEventType,
-} from 'react-native-screens';
+import React from 'react';
+import { Screen, ScreenProps } from 'react-native-screens';
 
 // @ts-ignore file to be used only if `react-native-reanimated` available in the project
-import Animated, { useEvent, useSharedValue } from 'react-native-reanimated';
-import ReanimatedTransitionProgressContext from './ReanimatedTransitionProgressContext';
+import Animated from 'react-native-reanimated';
 
 const AnimatedScreen = Animated.createAnimatedComponent(
   (Screen as unknown) as React.ComponentClass
 );
 
-class ReanimatedScreenWrapper extends React.Component<ScreenProps> {
-  private ref: React.ElementRef<typeof View> | null = null;
-
-  setNativeProps(props: ScreenProps): void {
-    this.ref?.setNativeProps(props);
-  }
-
-  setRef = (ref: React.ElementRef<typeof View> | null): void => {
-    this.ref = ref;
-    this.props.onComponentRef?.(ref);
-  };
-
-  render() {
-    return (
-      <ReanimatedScreen
-        {...this.props}
-        // @ts-ignore some problems with ref
-        ref={this.setRef}
-      />
-    );
-  }
-}
-
 const ReanimatedScreen = React.forwardRef<typeof AnimatedScreen, ScreenProps>(
   (props, ref) => {
-    const { children, ...rest } = props;
-
-    const progress = useSharedValue(0);
-    const closing = useSharedValue(0);
-    const goingForward = useSharedValue(0);
-
     return (
       <AnimatedScreen
         // @ts-ignore some problems with ref and onTransitionProgressReanimated being "fake" prop for parsing of `useEvent` return value
```

Comment:
Could you check if those `@ts-ignore` are still needed? Maybe we can get rid of them now.

Reply:
It is still required